airflow/config_templates/default_airflow.cfg

```diff
@@ -70,6 +70,7 @@ simple_log_format = %%(asctime)s %%(levelname)s - %%(message)s
 # we need to escape the curly braces by adding an additional curly brace
 log_filename_template = {{{{ ti.dag_id }}}}/{{{{ ti.task_id }}}}/{{{{ ts }}}}/{{{{ try_number }}}}.log
 log_processor_filename_template = {{{{ filename }}}}.log
+dag_processor_manager_log_location = {AIRFLOW_HOME}/logs/dag_processor_manager/dag_processor_manager.log
```

Comment:
For consistency can we call this `log_dag_processor_manager_location` (i.e. so that it starts with `log_` like the two before it.

Reply:
lol, I started with `log_dag_processor_manager_location` and then switched back because it reads bit weird to me... (`log_filename_template` reads good but `log_prodcessor_file_template` also reads weird to me, wondering if `log_prodcessor_file_template` was just named because the author saw the previous template config starts with log) I would actually even prefer to rename `log_processor_filename_template ` to make the names more intuitive to understand.